unity-renderer/Assets/UIComponents/Tests/ToggleComponentViewTests.cs

```diff
@@ -0,0 +1,94 @@
+using NUnit.Framework;
+using UnityEngine;
+
+public class ToggleComponentViewTests
```

Comment:
@AjimenezDCL is this naming something standarize by us? I'm using the "_____Tests" way in all our UI Components tests.

Reply:
Oh, we kinda agreed to it last year or so but it has not been written in stone anywhere, I leave it up to you Davide, keep the UIComponents convention or the project-wide one, it's fine anyway.

Reply:
I firstly wrote it as ___Tests because it was the same as the button component one, but at the same time the world Tests is redundant and doesn't give that much info, I kinda prefer the "Should" because it forces the tests to be fluent.

Update: just talked with Santi, will leave the ___Tests to fit the rest of UIComponents

Reply:
Yeah, for now I prefer to keep these tests with the same namings convention as the rest of them are following the other way. But we can rename all of them in another future PR in order to standarize it 👍